actors/src/main/kotlin/ktx/actors/events.kt

```diff
@@ -88,6 +89,20 @@ inline fun <T : Actor> T.onClickEvent(
   return clickListener
 }
 
+/**
+ * Attaches a [ClickListener] to this tree.
+ * @param lsitener invoked each time a node in this tree is clicked. Consumes the triggered [InputEvent] and the
+ * [Tree] the listener was originally attached to as `this`. The received Node is the node that was clicked.
+ * @return [ClickListener] instance.
+ */
+inline fun <T : Tree<*, *>> T.onItemClick(crossinline listener: ((Tree.Node<*, *, *>) -> Unit)): ClickListener {
+  val clickListener = object : ClickListener() {
```

Comment:
Tree's internal `ClickListener` seems to fire a change listener when the selection changes. It might make sense to use a change listener instead, but make sure to rename the entire function (e.g. `onSelectionChange`).